### ISSUE TITLE
[I2] Duda divisores del 0 en primos.

### ISSUE THREAD

**Issue by VicenteMerino** [marceloarenassaavedra/IIC2283-2-21#52]
Hola, estaba haciendo la pregunta 1 de la I1 del año pasado y creo que logré demostrar una dirección, sin embargo no se me ocurre la otra. Si tienen un tip de como empezar ese lado, lo agradecería mucho:

![image](https://user-images.githubusercontent.com/47841749/142502951-d09a3bed-3598-4a7b-967c-5af4aec24f8a.png)

![image](https://user-images.githubusercontent.com/47841749/142502902-1ce20478-2a9e-4926-9325-2b1a8be83dac.png)

**Comment by N9199** [comment]
Demuestra que si n no es primo entonces tiene divisores de 0.

**Comment by VicenteMerino** [comment]
Pregunta, p -> q, es lo mismo que no p -> no q?

**Comment by N9199** [comment]
Nope,p->q es equivalente a !q->!p, i.e. contrapositiva. En este caso, quieres demostrar un ssi, ya demostraste que n primo -> no hay divisores de 0, falta demostrar no hay divisores de 0 -> n es primo, esto último es equivalente a demostrar n no es primo -> existen divisores de 0

**Comment by VicenteMerino** [comment]
Buena, me queda claro ahora! Gracias Nicholas
